src/components/drawers/worker-drawer/worker-edit/worker-civil-contract-work-norm-selector.component.tsx

```diff
@@ -52,12 +52,12 @@ export function WorkerCivilContractWorkNormSelector({
     return workerHoursAsNumber / requiredHours;
   }
 
-  function handleCivilTimeChange(
+  const handleCivilTimeChange = useCallback((
     event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
-  ): void {
```

Comment:
Not sure why, but there started to appear this warning
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md

I thought, that it is quite useful and decided to fix it instead of disabling.